packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -496,7 +496,27 @@ class VotesRepo extends AbstractRepo<"Votes"> {
     );
     return results.map(({ vote_id }) => vote_id);
   }
-
+  
+  async getVotesOnSamePost({userId, postId, excludedDocumentId}: {
+    userId: string,
+    postId: string,
+    excludedDocumentId: string,
+  }): Promise<DbVote[]> {
+    return await this.manyOrNone(`
+      SELECT
+        v.*
+      FROM
+        "Votes" v
+        "Comments" c
+      WHERE
+        v."cancelled" IS FALSE
```

Comment:
I don't think it matters in this case post-nullability migration, but in general I prefer `IS NOT TRUE` where `null` would be semantically equal to `FALSE`